wizard/core/files.py

```diff
@@ -7,18 +7,7 @@
 import pandas as pd
 import requests
 
-
-@dataclasses.dataclass
-class File:
-    name: str = ""
-    description: str | None = None
-    content_url: str = ""
-    encoding_format: str | None = None
-    sha256: str | None = None
-    df: pd.DataFrame | None = None
-
-    def __post_init__(self):
-        self.df = self.df.infer_objects()
+from .state import FileObject
```

Comment:
In the whole repo, we use absolute imports relative to the root (i.e., where the pyproject.toml is). Is it possible to stick to this style in the editor? (I guess we should write a style guide listing those rules)

Reply:
I spent a couple of hours trying to figure this out (if you look at some of my commit messages, I lamented how non-simple python imports are, compared to say, javascript).  If you can explain how to avoid relative imports here, I'd be glad to hear it, but i suspect if I spend more brain power trying to figure out how python imports work/are sane I'll forget something else important, like how to speak more than 3 words in a row without staring into space, or how to talk without drooling.